### DISCUSSION TITLE
close camera

### DISCUSSION THREAD

**Discussion by victorcode1** [juliansteenbakker/mobile_scanner#1039] [General]
I have noticed that if the device fails while the camera is on, when you start the app again, the device registers the camera as open and cannot be closed even with the close method of the controller, is there an automatic way to correct this?

**Comment by EArminjon** [comment]
Linked to https://github.com/juliansteenbakker/mobile_scanner/issues/1037 i believe ?

**Comment by victorcode1** [comment]
Yes but still the same problem
